### PR TITLE
NOD and SOC event statuses

### PR DESCRIPTION
**Pull request by bshyong**
Adds content/styling for NOD and SOC last event statuses

Uses latest content in these issues:
resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3215
resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3216

@NatalieEMoore can you confirm that the "we need your Form 9 by..." in the yellow box on the SOC status is **60 calendar days from SOC event date?**

### SOC
screen shot 2017-06-23 at 7 23 26 pm

### NOD
screen shot 2017-06-23 at 5 57 31 pm

### CODE REVIEW

**Comment by NatalieEMoore** [comment]
@bshyong that is correct.

**Comment by bshyong** [comment]
@U-DON can you give this a quick look?

**Review by U-DON — APPROVED** [review]
LGTM. Minor comments.

File: src/js/disability-benefits/containers/AppealStatusPage.jsx [hunk 1]
```diff
@@ -15,6 +15,60 @@ class AppealStatusPage extends React.Component {
...
+  renderStatusNextAction(lastEvent) {
+    const eventType = lastEvent.type;
+    const { nextAction } = appealStatusDescriptions[eventType];
+    const className = `next-action ${eventType}`;
+    let title = nextAction.title;
+
+    switch (eventType) {
+      case 'form9':
+        break;
+      case 'soc':
```
Comment (bshyong) [inline src/js/disability-benefits/containers/AppealStatusPage.jsx]: Not sure yet until I finish the other event types 😛

**Pull request merged**